spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/openai/OpenAiProperties.java

```diff
@@ -84,6 +94,14 @@ public void setBaseUrl(String baseUrl) {
 		this.baseUrl = baseUrl;
 	}
 
+	public String getEmbeddingApiKey() {
```

Comment:
how did the need for this come about?  Can one have a different API key for embedding vs generation/inference?

Reply:
My apologies for the confusion, but I rearranged the order of the state variable in the code and that is why this appears in my commit.  Here is the [original definition](https://github.com/spring-projects-experimental/spring-ai/blob/main/spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/openai/OpenAiProperties.java#L41-L45) of `OpenAiProperties`.

It was an organizational thing.